README.md

```diff
@@ -37,15 +37,15 @@ const { printTable } = require('console-table-printer');
 
 //Create a table
 const testCases = [
-  { index: 3, text: 'I would like some gelb bananen bitte', value: 100 },
-  { index: 4, text: 'I hope batch update is working', value: 300 },
+  { Rank: 3, text: 'I would like some Yellow', value: 100 },
+  { Rank: 4, text: 'I hope batch update is working', value: 300 },
```

Comment:
I'd like to suggest changing `Rank` to `rank` because most keys I see in JavaScript start with lowercase. Thanks.

Reply:
Thanks for the suggestion. I was about to create another PR for making this change. But then thought maybes its better because, having also some uppercase in dataset makes it look like "keys"/"Column Names" don't need to be Capitalized.

Reply:
> having also some uppercase in dataset makes it look like "keys"/"Column Names" don't need to start with lowercase

IMHO, given javascript syntax, I think that's already implied.